src/main/java/io/cdap/plugin/salesforce/plugin/sink/batch/SalesforceSinkConfig.java

```diff
@@ -170,6 +184,19 @@ public String getExternalIdField() {
     return externalIdField;
   }
 
+  public String getConcurrencyMode() {
+    return concurrencyMode;
+  }
+
+  public ConcurrencyMode getConcurrencyModeEnum() {
+    try {
+      return ConcurrencyMode.valueOf(concurrencyMode);
```

Comment:
See if we need null check here and if upgrade from previous version works fine.

Reply:
Returning the default value: ConcurrencyMode.Parallel if

    the value is null or
    this enum type has no constant with the specified name

Tested with pipeline upgrade and it returns 200 OK. No issues were observed.

293aea61e74789e15658e6ba39309865ab94b355

Reply:
Added error logger in the catch block. 23502c1717880e2be601337d0ce5ca58a32c3fc7

Reply:
Add it as an INFO message in the log, and also mention that Parallel is being used as default.